Documentation/matchbox.md

```diff
@@ -128,9 +128,9 @@ Group selectors can use any key/value pairs you find useful. However, several la
 * `hostname` - hostname reported by a network boot program
 * `serial` - serial reported by a network boot program
 
-### Config Templates
+### Config templates
 
-Profiles can reference various templated configs. Ignition JSON configs can be generated from [Fuze config](https://github.com/coreos/fuze/blob/master/doc/configuration.md) template files. Cloud-Config templates files can be used to render a script or Cloud-Config. Generic template files can be used to render arbitrary untyped configs (experimental). Each template may contain [Go template](https://golang.org/pkg/text/template/) elements which will be rendered with machine group metadata, selectors, and query params.
+Profiles can reference various template configs. Ignition JSON configs can be generated from [Fuze config](https://github.com/coreos/fuze/blob/master/doc/configuration.md) template files. Cloud-Config templates files can be used to render a script or Cloud-Config. Generic template files can be used to render arbitrary untyped configs (experimental). Each template may contain [Go template](https://golang.org/pkg/text/template/) elements which will be rendered with machine group metadata, selectors, and query params.
```

Comment:
i guess templated has some specific meaning here. my assumption was there are  templates for config files.